src/SEOTools/SEOMeta.php

```diff
@@ -186,6 +186,11 @@ public function generate($minify = false)
      */
     public function setTitle($title, $appendDefault = true)
     {
+
+    	if (false === $title) {
+            return;
```

Comment:
Sim, além disso precisa modifcar o titulo, para ficar como padrão
@marcellorg pode ajustar?
`$this->title = $this->getDefaultTitle();`